lib/ocpp/common/database_handler.cpp

```diff
@@ -323,6 +323,11 @@ boost::optional<v16::IdTagInfo> DatabaseHandler::get_authorization_cache_entry(c
         const auto expiry_date = DateTime(expiry_date_str);
         id_tag_info.expiryDate.emplace(expiry_date);
     }
+    // if expiry date is not set, calling has_value() on the returned optional will return true, but calling value()
```

Comment:
Calling has_value() on an boost:optional will return false if the value of the optional is not set, so setting id_tag_info.expiryDate = boost::none is not necessary